InternxtDesktop/Views/Backups/BackupConfigView.swift

```diff
@@ -121,10 +121,12 @@ struct BackupConfigView: View {
                 self.changeFolders()
             } else {
                 try await self.backupsService.startBackup(onProgress: {progress in })
+                EventsUtils.trackSuccessBackup(foldersToBackup: backupsService.foldersToBackup.count)
```

Comment:
Track this event from the XPCBackupService, this way the track is more reliable, if we reach this line, the backup has been completed successfully

https://github.com/internxt/drive-desktop-macos/blob/main/XPCBackupService/XPCBackupService.swift#L106